addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9757,9 +9758,10 @@ msgctxt "#19182"
 msgid "Days to display"
 msgstr ""
 
+#. Settings -> Interface -> Other -> Startup window
 #: system/settings/settings.xml
 msgctxt "#19183"
-msgid "Radio"
+msgid "Radio channels"
```

Comment:
Yep. It's exclusively used for Settings -> Interface -> Other -> Startup window value. Checked xbmc core (incl. Estuary and Estouchy) + Confluence.